src/react-apps/applications/ux-editor/package.json

```diff
@@ -42,7 +42,9 @@
     "reselect": "3.0.1",
     "uuid": "3.2.1",
     "webpack": "4.12.0",
-    "webpack-dev-server": "3.1.14"
```

Comment:
this is not re-added here, but it's done for runtime.. Do we not need it here, @lorang92 ?